TROUBLESHOOTING.rst

```diff
@@ -135,8 +135,6 @@ because that only installs the interface to MySQL.
 Issues with pyodbc
 ------------------
 
-.. TODO: Add link to ODBC driver installation instructions once we move to Read The Docs.
-
```

Comment:
This is a tangential fix, but I deleted this comment because I didn't think it was needed in readthedocs since all the contributing info is on one page and you can simply run CTRL+F ODBC.